apps/sasquatch/src/androidTest/java/com/microsoft/azure/mobile/sasquatch/activities/SettingsActivityTest.java

```diff
@@ -0,0 +1,203 @@
+package com.microsoft.azure.mobile.sasquatch.activities;
```

Comment:
Ok what's the point of all this then?
On Xamarin we have UI testing that actually use and verify Mobile Center SDK APIs (thanks to label checking, we update a label and assert on that).